### PR TITLE
Add Example

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
This is wrt #11.

I've been running tons of experiments (**960 to be exact**) on all of the aliases from the optax library. So far I've been able to optimize using various sets of learning rates, weight decay and batch sizes.

I had some school work to do and haven't been able to work with the most custom features of the library (schedulers, second orders,.. ) That would be the next step. Hopefully, in the next PR, I'd be able to add some more examples using `measure_valued_jacobians` and `control_variates_jacobians`

### CODE REVIEW

**Comment by SauravMaheshkar** [comment]
Will Open a new PR.
